imports/ui/Accordian.jsx

```diff
@@ -9,6 +9,7 @@ export default class Accordian extends Component{
   getRandNum(){
     return Math.floor((Math.random() * 1000000) + 0);
   }
+
   render(){
     let accId = "accordion"+this.getRandNum();
```

Comment:
Using a random number generator to come up with an ID can lead to insecurities, especially with an increasing number of ID's; You can use an alternate solution to ensure that you don't result in collisions.
@julian-londono